doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,444 @@
+
+ >Note: The Windows Package Manager does not support loose executables with the .exe or .com file extension directly. Compressed files containing installers, or loose executables are also not supported.
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+
+</details>
+
+### Scope
+<details>
+ <summary>Scope indicates if the installer is per user or per machine</summary>
+
+ #### Optional Field
+
+ This key represents the scope the package is installed under. The two configurations are "user" and "machine". Some installers support only one of these scopes while others support both via arguments passed to the installer using "InstallerSwitches".
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+
+</details>
+
+### InstallModes
+<details>
+ <summary>List of supported installer modes</summary>
+
+ #### Optional Field
+
+ This key represents the install modes supported by the installer. The Microsoft community package repository requires a package support "silent" and "silent with progress". The Windows Package Manager also supports "interactive" installers.
+
+ >Note: Some installers will attempt to install missing dependencies. If these dependencies require user interaction, the package will not be allowed into the Microsoft community package repository.
+
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+</details>
+
+### InstallerSwitches
+<details>
+ <summary>Silent is the value that should be passed to the installer when user chooses a silent or quiet install</summary>
+
+ #### Optional Field
+
+ This key represents switches passed to the installer to provide a silent install experience.
+
+ >Note: When the Windows Package Manager installs a package using the "silent" install mode, any custom switches will also be passed to the installer. If a user applies override switches via command line via the Windows Package Manager, none of the switches from the manifest will be passed to the installer.
```

Comment:
Custom switch is always passed to installer regardless of "silent" or "non-silent" install, unless user provides Override using cmdline args which overrides all switches.

Reply:
Never mind, I misread the notes.